packages/models/package.json

```diff
@@ -32,12 +32,15 @@
     "@ckb-js/kuai-common": "0.0.1-alpha.0",
     "@ckb-js/kuai-io": "0.0.1-alpha.0",
     "@ckb-lumos/base": "0.20.0",
+    "@ckb-lumos/bi": "0.20.0",
     "@ckb-lumos/codec": "0.20.0",
     "@ckb-lumos/experiment-tx-assembler": "0.20.0",
+    "@ckb-lumos/lumos": "0.20.0",
```

Comment:
we have added @ckb-lumos/lumos at the 38 line, other lumos dependencies may be unnecessary. Because lumos has include others package.